lib/template/spec/endpoints/health_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+
+RSpec.describe Endpoints::Health do
+  include Rack::Test::Methods
+
+  def app
+    Endpoints::Health
+  end
+
+  describe 'GET /health' do
+    it 'returns a 200' do
+      get '/health'
+      assert_equal(200, last_response.status)
+      assert_equal('application/json;charset=utf-8', last_response.headers['Content-Type'])
+      assert_equal(2, last_response.headers['Content-Length'].to_i)
+      assert_equal({}, MultiJson.decode(last_response.body))
+    end
+  end
+
+  describe 'GET /health/db' do
+    it 'raises a 404 when no database is available' do
+      allow(DB).to receive(:nil?).and_return(true)
+
+      assert_raises Pliny::Errors::NotFound do
+        get '/health/db'
+      end
+    end
+
+    it 'raises a 503 on Sequel exceptions' do
+      allow(DB).to receive(:test_connection).and_raise(Sequel::Error)
+
+      assert_raises Pliny::Errors::ServiceUnavailable do
+        get '/health/db'
+      end
+    end
+
+    it 'raises a 503 when connection testing fails' do
+      allow(DB).to receive(:test_connection).and_return(false)
+
+      assert_raises Pliny::Errors::ServiceUnavailable do
+        get '/health/db'
+      end
+    end
+
+    it 'returns a 200' do
+      connection = double('connection', test_connection: true)
+      allow(Sequel::DATABASES).to receive(:first).and_return(connection)
```

Comment:
Would it make sense to stub `DB.test_connection` as it's already done [here][1] and [here][2]?

```ruby
allow(DB).to receive(:test_connection).and_return(true)
```

Btw, I :heart: the idea of `/health` endpoint! Thank you @gudmundur!

[1]: https://github.com/gudmundur/pliny/blob/e75a74bf923b2ec229e91075e56ef03b73ea113c/lib/template/spec/endpoints/health_spec.rb#L30
[2]: https://github.com/gudmundur/pliny/blob/e75a74bf923b2ec229e91075e56ef03b73ea113c/lib/template/spec/endpoints/health_spec.rb#L38